sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -1506,7 +1506,7 @@ private ServiceBusAsyncConsumer getOrCreateConsumer() {
                 .addKeyValue(LINK_NAME_KEY, linkName)
                 .addKeyValue(ENTITY_PATH_KEY, next.getEntityPath())
                 .addKeyValue("mode", receiverOptions.getReceiveMode())
-                .addKeyValue("isSessionEnabled", CoreUtils.isNullOrEmpty(receiverOptions.getSessionId()))
+                .addKeyValue("isSessionEnabled", !CoreUtils.isNullOrEmpty(receiverOptions.getSessionId()))
```

Comment:
It sees clearer to use `receiverOptions.isSessionReceiver()` to check? any reason we choose to use `sessionId` to check?

Reply:
That's a good idea! I'll make the change